qt_ui/windows/AirWingConfigurationDialog.py

```diff
@@ -163,6 +197,11 @@ def on_base_changed(self, index: int) -> None:
     def reset_title(self) -> None:
         self.setTitle(f"{self.squadron.name} - {self.squadron.aircraft}")
 
+    def reroll_nickname(self) -> None:
+        self.nickname_edit.setText(
+            self.squadron.coalition.air_wing.squadron_def_generator.random_nickname()
```

Comment:
Was thinking the same but actually it could also seen as a "feature": who wants to reroll the same nickname? :D I think its fine for the time beeing as its only used during campaign generation. Hopefully no one will click it 100k times. but before a minor performance decrease will hit in as it's always checking the already uses nicknames :D